### ISSUE TITLE
Getting HTTP 500 on 4.8.0

### ISSUE THREAD

**Issue by LeCodeCo** [phpmyadmin/phpmyadmin#14187]
### Steps to reproduce
1. Download 4.8.0 zip
2. Extract and enter IP of host in config
3. Try to log in and get PHP 500

### Expected behaviour
It should log in and display PhpMyAdmin

### Actual behaviour
Displays PHP 500 error.
https://file.properties/paste/azudafabut
https://i.imgur.com/EPH0NlJ.png

### Server configuration
**Operating system**:
Ubuntu 16.04

**Web server:**
Nginx

**Database:**
MariaDB

**PHP version:**
7.2.4-1+ubuntu16.04.1+deb.sury.org+1

**phpMyAdmin version:**
4.8.0

### Client configuration
**Browser:**
Google Chrome

**Operating system:**
Windows 10

**Comment by qcybb** [comment]
It almost appears that your phpmyadmin table was renamed or deleted.

This is a shot in the dark, but add `$cfg['ZeroConf'] = false;`

**Comment by ibennetch** [comment]
Pasting the error here from the linked pastebin:


```
2018/04/12 01:05:48 [error] 17869#17869: *72168 FastCGI sent in stderr: "PHP message: PHP Fatal error:  Uncaught Error: Call to a member function exists() on null in /var/www/html/app/public/php/libraries/classes/DatabaseInterface.php:1544
Stack trace:
#0 /var/www/html/app/public/php/libraries/classes/DatabaseInterface.php(2477): PhpMyAdmin\DatabaseInterface->postConnectControl()
#1 /var/www/html/app/public/php/libraries/common.inc.php(358): PhpMyAdmin\DatabaseInterface->connect(257)
#2 /var/www/html/app/public/php/index.php(26): require_once('/var/www/html/p...')
#3 {main}
  thrown in /var/www/html/app/public/php/libraries/classes/DatabaseInterface.php on line 1544" while reading response header from upstream, client: *.*.*.*, server: *.*.com, request: "POST /php/index.php HTTP/1.1", upstream: "fastcgi://unix:/var/run/php/php7.2-fpm.sock:", host: "*.*.com"
```

**Comment by williamdes** [comment]
Fixed in 4.8.1 by https://github.com/phpmyadmin/phpmyadmin/commit/8af0343bb288784e4c5bef91a2d84c1df5b97d94

**Comment by devenbansod** [comment]
Closing this as this seems to be fixed by @williamdes's patch in https://github.com/phpmyadmin/phpmyadmin/commit/8af0343bb288784e4c5bef91a2d84c1df5b97d94

@LeCodeCo feel free to re-open if this has not been fixed.
